src/lab7-tests.md

```diff
@@ -60,9 +60,9 @@ has the same total height:
     >>> browser2.load(url)
     >>> lab6.print_tree(browser2.document)
      DocumentLayout()
-       BlockLayout(x=13, y=18, width=774, height=60.0)
-         BlockLayout(x=13, y=18, width=774, height=60.0)
-           InlineLayout(x=13, y=18, width=774, height=60.0)
+       BlockLayout(x=13, y=18, width=774, height=45.0)
```

Comment:
Ditto.

Reply:
This one is due to adding `BlockLayout`.

Reply:
..and therefore fixing a bug in that InlineLayout was wrong.